README.md

```diff
@@ -18,6 +18,23 @@ Jetty is a modern fully async web server that has a long history as a component
 
 - [https://projects.eclipse.org/projects/rt.jetty](https://projects.eclipse.org/projects/rt.jetty)
 
+Webapp Example
+--------------
+```bash
+# java -jar $JETTY_HOME/start.jar --add-modules=http,deploy
```

Comment:
Change `#` to `$` as it's more traditional for user-space shell (not everyone lives on the dangerous road like you and use root all of the time)

Do we want some preface here?

``` shell
$ mkdir mybase
$ cd mybase
$ java -jar $JETTY_HOME/start.jar --add-modules=http,deploy
$ cp ~/src/myproj/target/mywebapp.war webapps
$ java -jar $JETTY_HOME/start.jar
```